src/middleware/packages/backup/index.js

```diff
@@ -2,28 +2,31 @@ const { CronJob } = require('cron');
 const fsCopy = require('./utils/fsCopy');
 const ftpCopy = require('./utils/ftpCopy');
 const rsyncCopy = require('./utils/rsyncCopy');
+/**
+ * @typedef {import('moleculer').Context} Context
```

Comment:
Why do you need to define this ? It's not used below.

Reply:
My bad! That was an artifact from older changes.